### ISSUE TITLE
Broken API

### ISSUE THREAD

**Issue by JamesCullum** [JamesCullum/ProtonMail-Checker#5]
I just sent an email to ProtonMail due to user reports of a broken plugin.
I can confirm that the /auth backend doesn't return the required data anymore.

@bartbutler do you know more about this? You used to update me about this issue and helped before. 

In addition, do you have any idea on how the popup - UI could be working without having to "remote control" it? Its super slow and requires changes whenever you roll out new versions.

**Comment by bartbutler** [comment]
Hi @JamesCullum, yes, we dropped support for the old password API when we were able to get all the official clients using the new SRP scheme (https://protonmail.com/blog/encrypted_email_authentication/). Sorry I didn't let you know, it slipped my mind. You are welcome to implement it, but it's significantly more complicated than that old login procedure. Is there still a need for this project now that desktop notification are an option in the ProtonMail web client?

**Comment by JamesCullum** [comment]
Do the desktop notifications work while the website is closed without compromising the keys?

**Comment by bartbutler** [comment]
No, they only work when the a tab is open, which is a weakness I suppose. Anyway, I'm happy to supply you with the new documentation, but it is significantly more complicated than the old system unfortunately.

**Comment by JamesCullum** [comment]
There was already a chrome extension that was only working when the tab was open, and for me this is not how I want to use an email client. Thanks for your support, I will probably go for a workaround again. 

Would it be possible to let the user log onto the page and internally dump all session variables after login? This way I would try to restore the session inside the extension and keep it running in the background, not having to deal with too many internal stuff.
